inbm/dispatcher-agent/dispatcher/source/source_command.py

```diff
@@ -99,7 +99,7 @@ def _handle_app_source_command(
```

Comment:
Refactor the `_handle_app_source_command` function to reduce its complexity and improve readability. [important]